wordpress-mobile/WordPress-Android
# Fix/fixing some talkback issues

This PR fixes some accessibility issues.

### To test:
Enable talkback and check the following (to enable/disable talkback you can keep pressed both +/- volume buttons):
- Go to list of likers in the reader and check that talkback announces "List of likes <n> likes"
- Open a user profile bottom sheet and check the description `User profile bottom sheet information` is announced
- In the user profile bottom sheet check that the user bio is detected and read
- In the user profile bottom sheet the Site label is announced

## Regression Notes
1. Potential unintended areas of impact
N/A still behind feature flag.

PR submission checklist:
